README.md

```diff
@@ -74,7 +74,7 @@ The redirect parameter redirect the user to the corresponding url instead if set
 
 ### Dependencies
 
-The **Make** targets assume you have **bash**, **curl**, **tar**, **docker** and **docker-compose** installed.
+The **Make** targets assume you have **bash**, **curl**, **tar**, **docker** and **docker compose** installed.
```

Comment:
Wouldn't it be `docker-compose-plugin` instead of "docker and docker compose" here?
(at least according to https://docs.docker.com/compose/install/linux/)

Reply:
Yes indeed, thank you.